stats/measure.go

```diff
@@ -20,19 +20,33 @@ import (
 	"sync/atomic"
 )
 
-// Measure represents a type of metric to be tracked and recorded.
-// For example, latency, request Mb/s, and response Mb/s are measures
+// Measure represents a single numeric value to be tracked and recorded.
+// For example, latency, request bytes, and response bytes could be measures
 // to collect from a server.
 //
-// Each measure needs to be registered before being used.
-// Measure constructors such as Int64 and
-// Float64 automatically registers the measure
-// by the given name.
-// Each registered measure needs to be unique by name.
-// Measures also have a description and a unit.
+// Measures by themselves have no outside effects. In order to be exported,
+// the measure needs to be used in a View. If no Views are defined over a
+// measure, there is very little cost in recording it.
 type Measure interface {
+	// Name returns the name of this measure.
+	//
+	// Measure names are globally unique (among all libraries linked into your program).
+	// We recommend prefixing the measure name with a domain name relevant to your
+	// project or application.
+	//
+	// Measure names can be as long as you like, they are
+	// never sent over the wire or exported to backends in any way. They are only
+	// used to create Views.
 	Name() string
+
+	// Description returns the human-readable description of this measure.
+	// For documentation only.
```

Comment:
For documentation of what? This is very confusing, please remove.

Reply:
What would you suggest replacing it with? If it doesn't exist for documentation purposes, what does is it for?